api/db/conn.js

```diff
@@ -1,14 +1,23 @@
 import { MongoClient } from 'mongodb';
 
-const { MONGO_USERNAME, MONGO_PASSWORD, HOSTNAME, MONGO_PORT } = process.env;
-const client = new MongoClient(`mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${HOSTNAME}:${MONGO_PORT}`);
+const { MONGO_USERNAME, MONGO_PASSWORD, MONGO_PORT, MONGO_SERVICE, CONTAINERIZED } = process.env;
+
+// Changing where to look for Mongo
+const MONGO_HOSTNAME = !!CONTAINERIZED ? MONGO_SERVICE : 'localhost';
+
+// Create connection client
+const client = new MongoClient(`mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_HOSTNAME}:${MONGO_PORT}`);
+
+// Attempt connection
 let conn;
 try {
   conn = await client.connect();
+  console.log(`MongoDB connected on port ${MONGO_PORT}.`)
 } catch (e) {
   console.error(`Error connecting to MongoDB. \nDetails: ${e}`);
 }
 
+// Select database to use
 const db = conn.db(process.env.MONGO_DATABASE);
```

Comment:
This could be added to the destructured variables above for consistency. 

I will approve regardless as it is not breaking or critically important.

Reply:
Good point. I think I had this at some point before the latest round of Mongo issues. Must have gotten undone. Will include in future changes.